src/runner/Runner.java

```
@@ -1,29 +1,74 @@
 package runner;
 
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class Runner {
+	private final static String[] RESTAURANTS = { "Burger King", "La Place", "Vapiano", "Pizza Hut", "McDonald's",
+			"Subway", "KFC", "Sodexo" };
+	private static final String EXCLUDED = "Burger King|Sodexo";
+
 	public static void main(String[] args) {
-		String selection = restaurantSelector();
-		if (selection.equals("burger king")) {
-			System.out
-					.println("Unfortunately burger king has been selected, but since we hate those guys a retoss is being performed");
-			selection = restaurantSelector();
-			System.out.println("New selection is: " + selection);
-		} else {
-			System.out.println("Your place of dining today is: " + selection);
+		int count = 100;
+		if (args.length > 0) {
+			try {
+				count = Integer.parseInt(args[0]);
+			} catch (Exception e) {
```

I prefer to fail silently given the lack of importance of this potential issue.